File: Module/ALittleIDE/Other/GameLibrary/CEngine/JSScript/CEngine/Version/VersionSystemIOS.js
{
if (typeof ALittle === "undefined") ALittle = {};


if (ALittle.VersionSystem === undefined) throw new Error(" extends class:ALittle.VersionSystem is undefined");
ALittle.VersionSystemIOS = JavaScript.Class(ALittle.VersionSystem, {
	Ctor : function(account_name, module_name) {
		this._install_name = "Install.ipa";
	},
	RefreshVersion : function() {
		ALittle.VersionSystemAndroid.RefreshVersion();
	},
	Install : function(install_name) {
	},
}, "ALittle.VersionSystemIOS");

}

File: ALittle/Module/ALittleIDE/Other/GameLibrary/CEngine/Script/CEngine/Version/VersionSystemIOS.lua

module("ALittle", package.seeall)

local ___rawset = rawset
local ___pairs = pairs
local ___ipairs = ipairs


assert(ALittle.VersionSystem, " extends class:ALittle.VersionSystem is nil")
VersionSystemIOS = Lua.Class(ALittle.VersionSystem, "ALittle.VersionSystemIOS")

function VersionSystemIOS:Ctor(account_name, module_name)
	___rawset(self, "_install_name", "Install.ipa")
end

function VersionSystemIOS.RefreshVersion()
	VersionSystemAndroid.RefreshVersion()
end

function VersionSystemIOS:Install(install_name)
end

